terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraintvalidators/CompareValidator.java

```diff
@@ -52,6 +52,12 @@
     private Operator operator;
 
     /**
+     * {@code true} it's valid if left and right are both null, but it's invalid if either is null. otherwise it's valid even if
+     * either is null.

```

Comment:
@ikeyat 
Thanks for your comments.
I've reflected in the javadoc. Please show 668bc68f0b7624153796f2027c242c2ce017a3e8.